src/math/exp_fcn.c

```diff
@@ -217,4 +216,52 @@ int32_t sofm_exp_int32(int32_t x)
 	}
 	return ts;
 }
+
+#define ONE_Q20         Q_CONVERT_FLOAT(1.0, 20)	  /* Use Q12.20 */
+#define TWO_Q27         Q_CONVERT_FLOAT(2.0, 27)	  /* Use Q5.27 */
+#define MINUS_TWO_Q27   Q_CONVERT_FLOAT(-2.0, 27)	  /* Use Q5.27 */
```

Comment:
those definition are too simple, suggest add EXP_ to avoid potential conflicts.
or you may directly use existing definition:
https://github.com/thesofproject/sof/blob/main/src/math/decibels.c

put it to a header file.